vimeo-networking/src/main/java/com/vimeo/networking/Configuration.java

```diff
@@ -96,24 +97,13 @@ public Cache getCache() {
         return new Cache(cacheDirectory, cacheSize);
     }
 
-    /**
-     * @deprecated use {@link #saveAccount(VimeoAccount, String)} instead
-     * <p/>
-     * We find no use in storing the password when you can persist the {@link VimeoAccount} across
-     * application sessions.
-     */
-    @Deprecated
-    public void saveAccount(VimeoAccount account, String email, String password) {
-        saveAccount(account, email);
-    }
-
-    public void saveAccount(VimeoAccount account, String email) {
+    public void saveAccount(@NotNull VimeoAccount account, String email) {
```

Comment:
May be null if this doesn't represent an authenticated user. I will add javadoc.